blist.js

```diff
@@ -1,170 +1,176 @@
 // Dependencies
 const axios = require("axios")
 const express = require("express")
-const d = require("discord.js")
-var bodyParser = require('body-parser')
-
-// Constants
+const discord = require("discord.js")
 const url = "https://blist.xyz"
 
+let bodyParser = require('body-parser')
+
 class Blist {
-    constructor(client, key) {
-        if (!key) {
-            console.log("Some functions might not be useable because you did not provide an API key.")
+    constructor(client, apiKey) {
+        if (!apiKey) {
+            console.log("Some functions might not be useable because you did not provide an API key");
         } else {
-            this.key = key
-        }
+            this.apiKey = apiKey;
+        };
 
-        if (!client instanceof d.Client) {
-            console.log("Bot client is not a discord.js client.")
+        if (!client instanceof discord.Client) {
+            return console.log("The Client argument must be a discord.Client");
```

Comment:
```suggestion
            return console.error("The client argument must be a discord.Client");
```
The argument name in the error message shouldn't be capitalized. Also, please look into using `console.error` instead of `console.log`.